bokeh/models/layouts.py

```diff
@@ -40,10 +40,10 @@ class LayoutDOM(Model):
     the widget will be greyed-out, and not respond to UI events.
     """)
 
-    responsive = Responsive(help="""
-    The type of responsiveness for the item being displayed. Possible values are
-    ``"fixed"`` (or ``False``), ``"scale_width"`` (or ``True``),
-    ``"scale_height"``, ``"scale_both"``, ``"stretch_both"``.
+    sizing_mode = Enum(SizingMode, default="fixed", help="""
```

Comment:
I know I said it should be fixed on everything - but I mispoke - I didn't mean at this high level.